ui/modules/things/things.html

```diff
@@ -88,7 +88,7 @@ <h5>Things</h5>
                                 <div class="invalid-feedback"></div>
                             </div>
                             <div class="input-group input-group-sm mb-1">
-                                <label class="input-group-text">Definition</label>
+                                <label class="input-group-text">Definition template</label>
```

Comment:
I would stick to just "Definition", as it represents the definition of the thing (if you select a thing, the definition is filled into that field).

Reply:
Ok, agreed, I will undo this. The "template" mechanism with the dropdown at this place however does not have much to do with the "definition" in my opinion.